### PR TITLE
Log Search: Histogram + Log Levels 📊 

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

New Log Search functionality: Log Levels 📊  reflected in Histogram and Messages tab.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/search/get-started-with-search/search-page/histogram.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Navigate through a large volume of logs
+* Filter the relevant logs in their troubleshooting workflows
+
+<!--
+<details><summary>What are log levels?</summary>
+
+define warn, critical, etc.
+
+</details>
+-->
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-page/histogram.md:45]: @tarunk2 I'm thinking we should briefly define Sumo Logic log levels? I realized it's not documented anywhere.

Comment (tarunk2) [reply docs/search/get-started-with-search/search-page/histogram.md:45]: Makes sense. We detect 6 log levels out of the box: FATAL, ERROR, WARN, INFO, DEBUG and TRACE. If we are not able to find one of these log levels in a message, it is categorized under "OTHERS" bucket.

File: docs/search/get-started-with-search/search-page/histogram.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Log-Level pattern detection is automatic, meaning you do not need to parse log levels manually or write specific queries to see your distribution of error logs. Just execute a log search, and you'll see:
+
+<img src={useBaseUrl('img/search/get-started-search/search-page/log-level-legend.png')} alt="log-level-legend" />
+
+| Element | Description
+| :-- | :--
+| A | Histogram with stacked bars representing log level distribution over your search timeframe.
+| B | Interactive Histogram legend showing the log level that each color represents. Click on any label to isolate it in the Histogram and corresponding messages beneath that. In this example, the **ERROR** label has been clicked, so you'll see only error logs reflected.<br/><img src={useBaseUrl('img/search/get-started-search/search-page/log-level-error-filter.png')} alt="log-level-error-filter.png" /><br/>To select multiple log levels, use **shift + click**.
+| C | Log messages corresponding to Histogram panels indicating log level severity. Log level message severity Quickly identify the log level of each log message using the color-coded legend for each message.
```
Comment (tarunk2) [inline docs/search/get-started-with-search/search-page/histogram.md:55]: How about "Log level for this log message. Quickly identify the log level of each log message using the color-coded legend for each message."?

File: docs/search/get-started-with-search/search-page/histogram.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+| D | **Hide Log Levels**. Click this to hide log levels from the Histogram chart, however, the **log_level** column will still show in the Messages table, but log level color coding corresponding will be removed. To enable it again, just click **Show Log Levels**.
```
Comment (tarunk2) [inline docs/search/get-started-with-search/search-page/histogram.md:56]: We can replace "column" by "field".

Comment (tarunk2) [reply docs/search/get-started-with-search/search-page/histogram.md:56]: Also "log level color coding corresponding will be removed" -> "log level color coding from individual messages will be removed"

File: docs/search/get-started-with-search/search-page/histogram.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+| E | [Field Browser](/docs/search/get-started-with-search/search-page/field-browser) **log_level** filter. You can use this reserved field in your queries to perform advanced operations.
```
Comment (tarunk2) [inline docs/search/get-started-with-search/search-page/histogram.md:57]: We can add "F" that shows the distribution of log levels under "_loglevel" field in the field browser.

Comment (tarunk2) [inline docs/search/get-started-with-search/search-page/histogram.md:57]: Worth mentioning as a NOTE that "_loglevel" field value for the messages with "<OTHERS>" log level in the histogram is `null`. So to query these messages, users can run `| where isNull(_loglevel)`.

File: docs/search/get-started-with-search/search-page/histogram.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+* Press the **X** in the selection area to clear a selection.
+* Click elsewhere in the histogram to make another selection.
+* Press **Shift Click** in the selected area (pink highlighted area) to open a new search tab for the selected time range.
+* Your selection is maintained if you go to another tab, but not if you close or save the search.
+
+If you'd prefer not to display the Log Search histogram, click **Hide Histogram**.
+
+<img src={useBaseUrl('img/search/get-started-search/search-page/hide-histogram.png')} alt="Hide Histogram" width="300"/>
+
+## Filter by Log Level
```
Comment (tarunk2) [inline docs/search/get-started-with-search/search-page/histogram.md:30]: Same as above - this feature is going to be in beta. Should we indicate it and mention the steps to get access to it?

**Review by tarunk2 — APPROVED** [review]
LGTM, thanks!

**Comment by kimsauce** [comment]
> LGTM, thanks!

@tarunk2 thank you for the quick review! What time should I push this live?

**Comment by tarunk2** [comment]
@kimsauce Harrish has a feedback on this: https://jira.kumoroku.com/jira/browse/SUMO-209763?focusedCommentId=697974&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-697974

```
can we move this page under the beta section instead of the public help docs? This feature is not going to be available for all customers and we are explicitly approaching customers for beta recruitment.
```

**Comment by kimsauce** [comment]
> @kimsauce Harrish has a feedback on this: https://jira.kumoroku.com/jira/browse/SUMO-209763?focusedCommentId=697974&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-697974
> 
> ```
> can we move this page under the beta section instead of the public help docs? This feature is not going to be available for all customers and we are explicitly approaching customers for beta recruitment.
> ```

will do

File: sidebars.ts [hunk 6]
```diff
@@ -985,7 +985,7 @@ module.exports = {
           link: {type: 'doc', id: 'search/get-started-with-search/search-page/index'},
           items: [
             'search/get-started-with-search/search-page/add-saved-search-to-favorites',
-            'search/get-started-with-search/search-page/change-time-range-in-histogram',
+            'search/get-started-with-search/search-page/histogram',
```
Comment (kimsauce) [inline sidebars.ts:988]:
```suggestion
            'search/get-started-with-search/search-page/change-time-range-in-histogram',
```

File: cid-redirects.json [hunk 7]
```diff
@@ -1524,7 +1525,7 @@
   "/cid/10818": "/docs/integrations/sumo-apps/cse",
   "/cid/1082": "/docs/metrics/introduction/get-started-metrics",
   "/cid/10820": "/docs/manage/manage-subscription/create-manage-orgs",
-  "/cid/10821": "/docs/search/get-started-with-search/search-page/change-time-range-in-histogram",
+  "/cid/10821": "/docs/search/get-started-with-search/search-page/histogram",
```
Comment (kimsauce) [inline cid-redirects.json:1528]:
```suggestion
  "/cid/10821": "/docs/search/get-started-with-search/search-page/change-time-range-in-histogram",
```

File: cid-redirects.json [hunk 8]
```diff
@@ -336,7 +336,8 @@
   "/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page/Field-Browser": "/docs/search/get-started-with-search/search-page/field-browser",
   "/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page/Field-Browser/Search-from-the-Field-Browser": "/docs/search/get-started-with-search/search-page/field-browser/search-from-field-browser",
   "/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page/Field-Browser/Show-and-Hide-Fields-in-the-Field-Browser": "/docs/search/get-started-with-search/search-page/field-browser/show-hide-fields-in-field-browser",
-  "/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page/Change-the-Time-Range-in-the-Histogram": "/docs/search/get-started-with-search/search-page/change-time-range-in-histogram",
+  "/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page/Change-the-Time-Range-in-the-Histogram": "/docs/search/get-started-with-search/search-page/histogram",
+  "/docs/search/get-started-with-search/search-page/change-time-range-in-histogram": "/docs/search/get-started-with-search/search-page/histogram",
```
Comment (kimsauce) [inline cid-redirects.json:340]:
```suggestion
  "/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page/Change-the-Time-Range-in-the-Histogram": "/docs/search/get-started-with-search/search-page/change-time-range-in-histogram",
```

**Pull request merged**